x-pack/plugins/security/public/management/roles/edit_role/collapsible_panel/collapsible_panel.tsx

```diff
@@ -4,6 +4,8 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import './_collapsible_panel.scss';
```

Comment:
When doing this style of import, you can remove the `_`, which should only be used for partials. In this case, the sass is now the source file that is being compiled. You'll want to do this for all the files.

Reply:
TIL, thanks, will fix!